Packs/Base/Scripts/GetIndicatorsByQuery/GetIndicatorsByQuery.py

```diff
@@ -88,11 +88,16 @@ def find_indicators_with_limit_loop(indicator_query: str, limit: int):
     Finds indicators using while loop with demisto.searchIndicators, and returns result and last page
     """
     iocs: List[dict] = []
-    search_indicators = IndicatorsSearcher(query=indicator_query, limit=limit, size=PAGE_SIZE)
+    search_indicators = IndicatorsSearcher(
+        query=indicator_query,
+        limit=limit,
+        size=PAGE_SIZE,
+        filter_fields=",".join(populate_fields) if populate_fields and not unpopulate_fields else None,
```

Comment:
better use ArgToList
```suggestion
        filter_fields=argToList(populate_fields) if populate_fields and not unpopulate_fields else None,
```